src/sidebars/favourites/tearout-directive.js

```diff
@@ -108,9 +108,10 @@
                                         store = window.storeService.open(window.name);
                                     }
 
-                                    dragService.overAnotherInstance((overAnotherInstance) => {
+                                    dragService.overAnotherInstance('.favourites', (overAnotherInstance) => {
                                         if (overAnotherInstance) {
                                             dragService.moveToOtherInstance(scope.stock);
+                                            dragService.destroy();
                                             store.remove(scope.stock);
                                             dragService = null;
```

Comment:
I don't think we need a `dragService = null;`, unless there was a reason for it when you wrote it? A long time ago, I know, so I'm not sure if you'll be ableo to remember